ibis/backends/tests/test_register.py

```diff
@@ -425,6 +425,9 @@ def test_register_garbage(con, monkeypatch):
 def test_read_parquet(con, tmp_path, data_dir, fname, in_table_name):
     pq = pytest.importorskip("pyarrow.parquet")
 
+    if con.name in ["oracle", "exasol"]:
+        pytest.skip("Skip Exasol and Oracle because of the global pytestmark")
```

Comment:
Hi @cpcloud There is a global pytestmark for `oracle` and `exasol`, not sure how to override it in this specific test.  I tried to skip it, if it is the one we want, I could skip it in the `test_read_parquet_glob` too.

Since we are adding `read_parquet()` for some other backends, looks like we need to remove some backend from the `notyet` mark for this test `test_register_garbage`

Reply:
Generally when this happens you have to mark every other test in the file with the same marker, and then remove the backends you want to pass from the individual test marker.